Modules/_cursesmodule.c

```diff
@@ -3916,13 +3927,18 @@ _curses_qiflush_impl(PyObject *module, int flag)
  * and _curses.COLS */
 #if defined(HAVE_CURSES_RESIZETERM) || defined(HAVE_CURSES_RESIZE_TERM)
 static int
-update_lines_cols(void)
+update_lines_cols(PyObject *module)
 {
     PyObject *o;
     PyObject *m = PyImport_ImportModuleNoBlock("curses");
     _Py_IDENTIFIER(LINES);
     _Py_IDENTIFIER(COLS);
 
+    PyObject *d = PyModule_GetDict(module);
+    if (d == NULL) {
+        return NULL;
```

Comment:
> return makes integer from pointer without a cast [-Wint-conversion]

wrong return type in here?

Reply:
@shihai1991 Sorry, this is not ready to review.  I don't have an environment to test curses (I use either windows or WSL for linux and neither can build/test it) so I am using the CI to test it.

Reply:
This all right. I suggest you can create a PR to your fork repo. The fork repo's PR can trigger the CI gate too.

Reply:
@shihai1991 I'll try that for the next PR which actually converts to multi-phase init.  It seems like this one is done.